README.md

```diff
@@ -1,56 +1,37 @@
 # `ayb`
-With `ayb`, all your base can finally belong to you. Move SQL for great justice.
```

Comment:
This is your first chance to grab a users attention.  I understand the pun with the name, but they might not.  Further, although I understand the pun, I still don't understand what AYB actually does for me.

Instead use this sentence to sell AYB as a solution for a problem readers will understand.

Reply:
Thank you for this suggestion! I'll implement a hybrid of the more straightforward explanation and the fun throwback.